arielpartners/capdash2-server
# updated test task names and removed unimplemented cucumber features

TO TEST:
rake test
see report in reports/unit/results/index.html
rake test_unit
see same report
rake test_feature
see report in reports/feature/results/index.html
also see report in reports/coverage/index.html
